docs/src/examples/components/Toolbar/Content/ToolbarExamplePopup.shorthand.tsx

```diff
@@ -37,13 +12,7 @@ const ToolbarExamplePopupShorthand = () => {
           icon: { name: 'highlight', outline: true },
           active: highlightOpen,
           popup: {
-            content: (
-              <HighlightPopup
-                onConfirm={() => {
-                  setHighlightOpen(false)
-                }}
-              />
-            ),
+            content: <p>This is a popup</p>,
```

Comment:
Why did we change these examples? I am pretty sure they were meaningful exactly because of the inputs inside them.